src/Rules/IRuleHelp.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.Core;
+
+namespace SonarLint.VisualStudio.Rules
+{
+    /// <summary>
+    /// Help data about a single rule, extracted using the Java plugin API
+    /// </summary>
+    public interface IRuleHelp
+    {
+        /// <summary>
+        /// Unique identifier for the rule e.g. S123
+        /// </summary>
+        string RuleKey { get; }
+
+        Language Language { get; }
+
+        /// <summary>
+        /// The HTML des
+        /// </summary>
+        string HtmlDescription { get; }
+    }
```

Comment:
Interface definitions with minimal class implementations and tests.

Reply:
Sounds good!